src/services/CommandsService.js

```
@@ -0,0 +1,28 @@
+import axios from "axios";
+
+const API_URL = process.env.REACT_APP_API_URL;
+const URL_COMMAND = "v1/comanda"
+
+export async function save(data) {
+    try {
+        console.log(data);
+        const response = await axios.post(`${API_URL}/${URL_COMMAND}`, data);
+        return response.data;
+    } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)